src/pages/[platform]/build-a-backend/data/data-modeling/add-fields/index.mdx

```diff
@@ -143,6 +169,23 @@ client.models.Post.create({
 });
 ```
 
+</InlineFilter>
+
+<InlineFilter filters={["swift"]}>
+
+Creating a new item client-side with the enum value:
+
+```swift
+let post = Post(
+    content: "hello",
+    privacySetting: .private)
+let createdPost = try await Amplify.API.mutate(request: .create(post)).get()
+```
+
+</InlineFilter>
+
+<InlineFilter filters={["javascript", "angular", "react-native", "react", "nextjs", "vue"]}>
```

Comment:
I found this section only relevant for the TS client, by having this inline filter, removes it from swift/android/flutter